polylogue/cli.py

```
@@ -637,6 +867,57 @@ def run_import_codex(args: argparse.Namespace, env: CommandEnv) -> None:
+            result = sync_codex_sessions(
+                base_dir=base_dir,
+                output_dir=out_dir,
+                collapse_threshold=collapse,
+                html=html_enabled,
+                html_theme=html_theme,
+                force=False,
+                prune=False,
+                diff=False,
+                sessions=None,
+            )
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)